bin/vcf2gvf.py

```diff
@@ -562,7 +563,7 @@ def find_sample_size(table, lineage):
 
     args = parse_args()
     with open(args.gene_positions) as fp:
-        GENE_POSITIONS_DICT = json.load(fp)
+        GENE_PROTEIN_POSITIONS_DICT = json.load(fp)
```

Comment:
The new file will have gene + protein coordinates - [constellations](https://github.com/cov-lineages/constellations/blob/main/constellations/data/SARS-CoV-2.json)